src/wise_psf.jl

```diff
@@ -0,0 +1,39 @@
+using FITSIO
+
+function wise_psf(band; sidelen=-1)
+
+  # don't believe any sensible normalization is in place, need to fix that ...
+
+  # could potentially add checks on the input values ...
+
+  # figure out where the PSF model FITS file is located
+  psf_dir = ENV["WISE_ETC"]
+
+  psf_fname_dict = [1=>"psf_coeff-w1.v1.fits",
+                    2=>"psf_coeff-w2.v1.fits",
+                    3=>"psf_coeff-big.fits",
+                    4=>"psf_coeff-taper-w4.fits" ]
```

Comment:
Usually preferable to use an array rather than a dict with consecutive integers as indexes, unless you're planning for other types of keys in the future